script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -499,22 +507,22 @@ Shining brightly in the midst of it all like a jewel―――
 [sha:22610d021e2aa29c7c73cd867c054be51ca3bf5a]{
 -- Page 836, Offset 7877.
 -- あまりの頭痛に目を背ける。
-I turn my eyes away from the excessive headache.
+My headache becomes too much to bear, and I tear my eyes away.
 }
 [sha:254fd88af877b59c9160ef4324a6740c983a6098]{
 -- Page 836, Offset 7878.
 -- アレはまずい。あの男の“<最中|さなか>”を<視|み>続けるのはまずい。
-That thing is bad news. It would be foolish to continue looking at that point in the man's center.
+That thing is bad news. I shouldn't stare at that point at the man's core.
 }
 [sha:ae2c73986deb06adf69685c3ef742ad6cf8331d9]{
 -- Page 836, Offset 7879.
 -- 太陽を直視すれば網膜が焼き付くように。
-It's like looking directly at the sun and burning your retinas.
+It's like how if you look directly at the sun you'll burn your retinas.
 }
 [sha:c5ca1fcdbfdb33e3894868186a8682cb382fbec6]{
 -- Page 836, Offset 7880.
 -- 覚悟もなしに<ア|・><レ|・>を視ては、この脳が破裂する。
-My brain will explode if I look at %{i}that thing%{/i} without being prepared.
+My brain will explode if I look at that %{i}thing%{/i} without being prepared.
```

Comment:
not a fan of the end of this sentence. Maybe
"without preparing myself first" or "without preparation"